entrypoint.sh

```diff
@@ -9,22 +9,22 @@ set -eu
 export OXYGEN_DEPLOYMENT_TOKEN="$INPUT_OXYGEN_DEPLOYMENT_TOKEN"
 export OXYGEN_COMMIT_MESSAGE="$INPUT_COMMIT_MESSAGE"
 export OXYGEN_COMMIT_TIMESTAMP="$INPUT_COMMIT_TIMESTAMP"
-export OXYGEN_WORKFLOW_ID="$INPUT_WORKFLOW_RUN_ID"
 export OXYGEN_SHOP_ID="$INPUT_SHOP_ID"
 export OXYGEN_STOREFRONT_ID="$INPUT_STOREFRONT_ID"
 export OXYGEN_BUILD_COMMAND="$INPUT_BUILD_COMMAND"
+# We're purposefully ignoring the workflow ID input. It is unnecessary since it is already available as an env var.
+# Since removing an input is a breaking change, it will be removed in the next version (v4)
+export OXYGEN_WORKFLOW_ID="${GITHUB_RUN_ID}_${GITHUB_RUN_ATTEMPT}"
```

Comment:
Weren't we adding a prefix?

Reply:
I figured it wasn't _necessary_, but we could if we wanted to.

Reply:
It's not needed, but it will make our next migration smoother